test/src/050-configrepo/main

```diff
@@ -89,12 +98,17 @@ cvmfs_run_test() {
 
   CVMFS_TEST_050_CONF_EDIT=1
 
+  # because of the different CVMFS_MOUNT_DIR, this cvmfs_mount fails,
+  #  but call it anyway to set up the config
   cvmfs_mount grid.cern.ch
-  ls /cvmfs/grid.cern.ch || return 21
+  mkdir grid.cern.ch
+  mount -t cvmfs grid.cern.ch $PWD/grid.cern.ch || return 21
```

Comment:
You're right.  I'm sorry, that must have been at the beginning of my testing when I was forgetting that the tests were supposed to be run as an unprivileged user.